ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -390,6 +390,21 @@ function OcmIntegrationForm({
                             }
                         />
                     </FormLabelGroup>
+                    <FormLabelGroup
+                        fieldId="cloudSource.skipTestIntegration"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <Checkbox
+                            label="Create integration without testing"
+                            id="cloudSource.skipTestIntegration"
+                            aria-label="skip integration test"
```

Comment:
Nit: since this component has an `id` matching the `for` attribute of a label, it will already have an accessible name of `"Create integration without testing"`. We could omit the `aria-label` prop here and below and the spoken screenreader text for the checkbox would be identical to the text that is visible in the page.